instrumentation/google.golang.org/grpc/otelgrpc/interceptor.go

```diff
@@ -319,6 +320,16 @@ func UnaryServerInterceptor(opts ...Option) grpc.UnaryServerInterceptor {
 		info *grpc.UnaryServerInfo,
 		handler grpc.UnaryHandler,
 	) (interface{}, error) {
+		var statusCode grpc_codes.Code
+		var startTime = time.Now()
+		var attr []attribute.KeyValue
```

Comment:
`attr` is only ever used within the `defer`. So it doesn't need to be global.

Reply:
I‘m a litte confused, attr is first assigned from `spanInfo` func, and then appended with status code attribute. So I think it is not only used within defer? 

```
name, attr := spanInfo(info.FullMethod, peerFromCtx(ctx))
```

Reply:
I moved defer after the attr is created by spaninfo

Reply:
Done